### PR TITLE
Copy TOTP after autofill on page load by default

### PR DESCRIPTION
**Pull request by eliykat**
## Objective

Follow-up from #1626, which allowed a user to enable "Copy TOTP after autofill on page load", using a new checkbox in Options.

However, this led to the confusing situation where:
* TOTP is automatically copied after user-triggered autofill (on by default)
* TOTP is not automatically copied after autofill on page load (off by default, but can be enabled)
* The "Disable Automatic TOTP Copy" control overrides all of the above

This PR simplifies this a lot by largely reverting the changes in #1626, and enabling TOTP copy on **all** autofill operations. This treats all autofill operations equally, regardless of how they are triggered. The user can still disable automatic TOTP copy using the "Disable Automatic TOTP Copy" setting.

Previously I believe there have been concerns about writing sensitive data to the clipboard without user intervention. However:
* users have to opt-in to autofill on page load, so I presume that if they are comfortable auto-filling login information without user intervention, then they are also comfortable copying their TOTP.
* navigating to the popup to copy the TOTP is just as much work as manually triggering autofill, so I can't think of a use case where the user would want to autofill on page load but then manually copy the TOTP.

There is a risk of credentials being scraped, but in my opinion this is inherent in the autofill on page load feature, and we already let the user make the decision to accept this risk. However, I do think that if this change is merged, we should flag it clearly in our announcements so that users are aware. 

## Code changes

* remove unneeded UI controls and storage strings from #1626 
* tell `doAutoFillActiveTab` to always copy the TOTP (the "Disable Automatic TOTP Copy" option is dealt with in the `doAutoFill` method so we don't have to worry about it here)

Requires jslib: https://github.com/bitwarden/jslib/pull/379

### CODE REVIEW

File: src/services/autofill.service.ts [hunk 1]
```diff
@@ -259,13 +259,10 @@ export default class AutofillService implements AutofillServiceInterface {
...
         }
 
-        const copyTotpOnAutoFill = await this.totpService.isAutoCopyOnAutoFillEnabled();
-        const shouldCopyTotp = fromCommand || copyTotpOnAutoFill;
-
         const totpCode = await this.doAutoFill({
             cipher: cipher,
             pageDetails: pageDetails,
-            skipTotp: !shouldCopyTotp,
+            skipTotp: false,
```
Comment (Hinton) [inline src/services/autofill.service.ts:265]: Could we remove this option? I don't believe it was used outside this function.

**Review by Hinton — APPROVED** [review]

**Comment by eliykat** [comment]
Hey @tomershvueli, just wanted to give you a heads up on this. Our internal QA processes sent this feature back for a redesign because the UX was too confusing - see the explanation in my post above. Unfortunately, this means that a lot of the work we did together has been reverted in favour of a much simpler solution. However, I think you can be satisfied that the feature will still be in the next release - we just had to take the long way on this one. Such is life! Thanks again for your help getting this feature in the product one way or another and please don't hesitate to contribute again in the future.

**Comment by tomershvueli** [comment]
Appreciate the update @eliykat! No worries about the UX refresh, I'm just happy to hear the feature itself is going to be released soon :)

**Comment by dennyhollick** [comment]
@eliykat any updates on when we might see this? It doesn't look like it made the last release?

**Comment by eliykat** [comment]
Hi @dennyhollick, this feature has been released. If you have Autofill on Page Load enabled, and it autofills an item with a TOTP code, then the TOTP code should automatically be copied to your clipboard. (There is a setting in Options to disable TOTP copy, which can override this, so just make sure that hasn't been ticked.)

If this isn't working, then it's a bug. Please open a new Issue here on Github and I'll look into it.

**Pull request merged**